From 01618531ccdd7dddaea0fe3f37a1d76b49674646 Mon Sep 17 00:00:00 2001
From: Peter Tworek <tworaz666@gmail.com>
Date: Tue, 24 Mar 2015 17:56:58 +0100
Subject: [PATCH 1/9] Make it possible to use glib message pump together with
 ozone

Currently setting use_ozone to 1 makes glib support disabled by default.
It does not mean however that the user may not with to enable glib
anyway. Passing use_ozone=1 use_glib=1 is a perfectly valid use case
that is only broken due to some small gyp issues. Fix it.

Signed-off-by: Peter Tworek <tworaz666@gmail.com>
---
 base/base.gyp  | 2 +-
 base/base.gypi | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/base/base.gyp b/base/base.gyp
index b9362b4..562dcbb 100644
--- a/base/base.gyp
+++ b/base/base.gyp
@@ -753,7 +753,7 @@
             'message_loop/message_pump_glib_unittest.cc',
           ]
         }],
-        ['use_ozone == 1', {
+        ['use_glib==0', {
           'sources!': [
             'message_loop/message_pump_glib_unittest.cc',
           ]
diff --git a/base/base.gypi b/base/base.gypi
index e0c5c7d..ed04b30 100644
--- a/base/base.gypi
+++ b/base/base.gypi
@@ -929,7 +929,7 @@
               'strings/string16.cc',
             ],
           },],
-          ['<(use_ozone) == 1', {
+          ['<(use_glib) == 0', {
             'sources!': [
               'message_loop/message_pump_glib.cc',
             ]
-- 
2.0.5

